components/otp-input/src/ReuseOTPInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import { HeadlessOTPInput } from "@locoworks/reusejs-react-otp-input";
+import { HeadlessOTPInputProps } from "./HeadlessOTPInput";
+
+export interface ReuseOTPInputProps extends HeadlessOTPInputProps {}
+
+const ReuseOTPInput: React.FC<ReuseOTPInputProps> = ({
+  value = "",
+  numInputs = 4,
+  onChange,
+  renderInput,
+  shouldAutoFocus = false,
+  inputType = "text",
+  renderSeparator,
+  placeholder,
+  containerStyle,
+  inputStyle,
+}) => {
+  const [otp, setOtp] = useState("");
+
+  return (
+    <div className="flex flex-col items-center gap-x-3 justify-center py-10 mt-10 border rounded bg-gray-50">
+      <HeadlessOTPInput
+        value={otp}
```

Comment:
How do you override styles here? What prop value is used and where is it passed?

Reply:
@ShreyDhyani  Which component we will use in applications(Wiredup)? Is it a Reuse component or Headless Component?

I thought we will use the Headless component directly in applications like Wiredup.